From 031a44c15a5698397d5407e58bedaa22a6c12185 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Thu, 15 Feb 2018 11:21:09 +0100
Subject: [PATCH] CMakeLists: Remove Error message for poky compiler

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f66411a..19ac88e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,8 +35,8 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
 	if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.6)
 		message(FATAL_ERROR "Clang version must be at least 3.6! Earlier versions don't support debug information for auto.")
 	endif()
-else()
-	message(FATAL_ERROR "You are using an unsupported compiler! Compilation has only been tested with Clang and GCC.")
+#else()
+#	message(FATAL_ERROR "You are using an unsupported compiler! Compilation has only been tested with Clang and GCC.")
 endif()
 
 # Compiler flags
-- 
2.7.4

